lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -763,13 +764,23 @@ spec = do
                     }
             in
                 x' === x .&&. show x' === show x
-        it "ApiWalletMigrationPostData" $ property $ \x ->
+        it "ApiWalletMigrationPostData lenient" $ property $ \x ->
             let
                 x' = ApiWalletMigrationPostData
                     { passphrase =
-                        passphrase (x :: ApiWalletMigrationPostData ('Testnet 0))
+                        passphrase (x :: ApiWalletMigrationPostData ('Testnet 0) "lenient")
                     , addresses =
-                        addresses (x :: ApiWalletMigrationPostData ('Testnet 0))
+                        addresses (x :: ApiWalletMigrationPostData ('Testnet 0) "lenient")
+                    }
```

Comment:
We can actually write this much more concisely with the `OverloadedLabels` extension:

```hs
         it "ApiWalletMigrationPostData lenient" $ property $
             \(x :: ApiWalletMigrationPostData ('Testnet 0) "lenient") ->
                 let x' = ApiWalletMigrationPostData
                         { passphrase = view #passphrase x
                         , addresses  = view #addresses  x
                         }
                 in
                 x' === x .&&. show x' === show x
```
This way, we can avoid repeating the type (which is quite long!).

Reply:
Related discussion here: https://input-output-rnd.slack.com/archives/GBT05825V/p1590144408053800